CONTRIBUTING.md

```diff
@@ -1,12 +1,56 @@
-## Where to ask Questions?
+# How to contribute
 
-Questions can be asked in form of issues in [project_name] repository:
-[Github project issues url]
+If you would like to help us and contribute to this project, we would be happy to do so. 
+However, some rules are to be followed in order to keep a maximum of coherence and stability within repo.  
```

Comment:
"However, some rules must be respected in order to keep a maximum of consistency and stability within the repo." ?